app/src/commonMain/kotlin/ui/SettingsScreen.kt

```diff
@@ -79,21 +86,22 @@ fun SettingsScreen(
         content = { paddingValues ->
             Column(
                 modifier =
-                    Modifier
-                        .padding(paddingValues)
-                        .verticalScroll(rememberScrollState())
-                        .fillMaxSize(),
+                Modifier
+                    .padding(paddingValues)
+                    .verticalScroll(rememberScrollState())
+                    .fillMaxSize(),
             ) {
                 SettingsHeaderElement(
                     header = Res.string.settings_general,
                 )
                 SettingsClickableElement(
                     title = stringResource(Res.string.settings_default_currency),
                     subtitle =
-                        viewModel.selectedCurrencyName.ifEmpty {
-                            stringResource(Res.string.settings_system_default)
-                        },
+                    viewModel.selectedCurrencyName.ifEmpty {
+                        stringResource(Res.string.settings_system_default)
+                    },
                     onClick = viewModel::onSelectCurrency,
+                    icon = Icons.Rounded.CurrencyExchange,
```

Comment:
Is there a reason you used the currency exchange icon instead of just the dollar sign here?

Reply:
Not really, can change that if you want

Reply:
Using `Paid` also doesn't look that great. So maybe the one you had was better fitting.